social-team.md

```diff
@@ -0,0 +1,72 @@
+# Social Team
+
+Node.js as a project has a Social Team, a joint team between the TSC and CommComm to which the project has delegated management of official social media.
+
+## Responsibilities
+
+The Social Team is tasked with maintaining the project social accounts. This includes:
+
+- content creation
+- amplification of our community
+  - sharing, retweeting, and so on
+- engagement with our community
+  - liking, replying, and so on
+- updating account metadata (like bio, avatar, header images, and so on)
+
+With regard to posts and community interaction, there is not requirement or entitlement - rather, posts and interactions are completely at the discretion of the Social Team's members.
+
+### Platforms 
+
+This team controls and maintains the various social platforms that Node.js has:
+
+- [Node.js on Twitter](https://twitter.com/nodejs)
+- [Node.js on Facebook](https://www.facebook.com/nodejsfoundation)
+- [Node.js on LinkedIn](https://www.linkedin.com/company/node.js-foundation/)
+
+### Access
+
+Social Team members will have full access to all [platforms](#platforms).
+
+### Expectations
+
+The basic expectations are that **all** Social Team members will have both a willingness and ability to...
+
+- spend time weekly engaging on various platforms, or communicate the inability to do so with other social team members and work with them to ensure there is *some* coverage
+- spend time working with other social team members on building out good content
+- spend time working work project committees, working groups, initiatives, and teams to build effective outward communication as needed
+
+### Repositories
+
+The Social Team will be responsible for and maintain the following:
+
+- [nodejs/tweet](http://github.com/nodejs/tweet): a repo leveraging [Twitter Together](https://github.com/gr2m/twitter-together) that enables project members to queue up tweets to [@nodejs](https://twitter.com/nodejs) from within GitHub.
+
```

Comment:
Does this support queuing up retweets as well?

Reply:
Quote tweeting is possible since the minimum implementation of this is simply including your text and then a link to the tweet, which will result in the tweet being quoted.

This is not currently implemented but there's an open discussion in the repo: https://github.com/gr2m/twitter-together/issues/101

Reply:
If we can't use what's described to request retweets we probably want to add  a section for that. I typically tweet out the minutes/recording for meetings and would like to be able to ask that the Node.js account retweet those.

Reply:
My perception is that is covered in L10-11 of this - I would consider that amplification of our community.

I also send DMs to the account a lot and while I personally have found that effective I'm trying to not be prescriptive about how the Social Team is going to operate in this document beyond basic enablement of the community.

Given that this is possible:

```text
Notes from today's TSC meeting 👀
<tweet link>
```

which will result in a quote tweet ([here's an example](https://twitter.com/nodejs/status/1247937279772983302?s=20) from @nodejs last week), plus the lines that assert amplification of our community is something the group is tasked with, do you feel adding an additional section requiring retweets of people under some specific conditions is necessary or a hard requirement?

Reply:
Given that context I think its ok as is.